docs/install.rst

```diff
@@ -43,6 +43,19 @@ Check the "tutor" package on Pypi: https://pypi.org/project/tutor. You will need
 
 .. _install_binary:
 
+
+Virtual Environment (Recommended)
```

Comment:
Let's add this in Requirements section

Reply:
I have moved this into the Requirements section